### PR TITLE
CRD Conversion Webhook support for OLM

### PR DESCRIPTION
**Pull request by simrandhaliw**
**Description of the change:**
This PR adds a new field called ConversionCRDs to WebhookDescriptions to enable conversion webhook support for OLM

**Motivation for the change:**
To add CRD conversion webhook support to OLM. Further details can be found in this enhancement doc: https://github.com/operator-framework/enhancements/pull/30

### CODE REVIEW

File: pkg/operators/v1alpha1/clusterserviceversion_types.go [hunk 1]
```diff
@@ -180,6 +180,7 @@ type WebhookDescription struct {
 	AdmissionReviewVersions []string                                        `json:"admissionReviewVersions"`
 	ReinvocationPolicy      *admissionregistrationv1.ReinvocationPolicyType `json:"reinvocationPolicy,omitempty"`
 	WebhookPath             *string                                         `json:"webhookPath,omitempty"`
+	ConversionCrds          []string                                        `json:"conversionCrds,omitempty"`
```
Comment (njhale) [inline pkg/operators/v1alpha1/clusterserviceversion_types.go:183]: Acronyms should be capitalized for go fields: `ConversionCRDs`

File: crds/operators.coreos.com_clusterserviceversions.yaml [hunk 2]
```diff
@@ -8704,6 +8704,10 @@ spec:
                     containerPort:
                       type: integer
                       format: int32
+                    conversionCrds:
```
Comment (ecordell) [inline crds/operators.coreos.com_clusterserviceversions.yaml:8707]: this should be `conversionCRDs` according to the naming conventions (see section on acronyms: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions)

**Pull request merged**